### ISSUE TITLE
api.SharedStorage/Fence* - Shared Storage API &  Fenced Frames API & Protected Audiences API may not supported in webview

### ISSUE THREAD

**Issue by skyclouds2001** [mdn/browser-compat-data#24690]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

viewing https://source.chromium.org/chromium/chromium/src/+/main:android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt, I notice the three APIs (also a `ServiceWorkerRegistration.periodicSync` property, the Periodic Sync API itself is disabled in webview) is in the file, but mark as supported in webview

* Shared Storage API
  * https://chromestatus.com/feature/6256348582903808 also says not supported in webview
* Fenced Frames API
  * https://chromestatus.com/feature/5699388062040064 also says not supported in webview
* Protected Audiences API
  * The same, but it's data is very raw.

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

mark those features as not-supported

### Did you test this? If so, how?

no, simply check using chromium source code

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://github.com/chromium/chromium/blob/main/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt
https://github.com/chromium/chromium/blob/main/third_party/blink/renderer/platform/runtime_enabled_features.json5

**Comment by SamB** [comment]
> ### What information was incorrect, unhelpful, or incomplete?
> viewing https://source.chromium.org/chromium/chromium/src/+/main:android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt, I notice some feature should not be marked as supported in webview, but it is incorrectly marked as yes

HMM, looks quite machine-friendly: I wonder if it would be practical to extract the information mechanically from the git history?

**Comment by queengooborg** [comment]
The file from Chrome's you mentioned had been taken into consideration years ago when most updates to BCD were still performed manually.  Now, almost all data is collected by https://mdn-bcd-collector.gooborg.com automatically.

If BCD reports that something in WebView is supported but it's in that file, it's either because:
- It's actually supported after all, so that source file is wrong
- It's incorrectly exposed and overrides need to be added to the collector

Do you have access to WebView on Android?  If so, could you test which features are actually unsupported?

**Comment by skyclouds2001** [comment]
> The file from Chrome's you mentioned had been taken into consideration years ago when most updates to BCD were still performed manually. Now, almost all data is collected by https://mdn-bcd-collector.gooborg.com automatically.

Yes, as I later made some check, most old APIs seems to be correct, but some newly added API (like Shared Storage API, Fenced Frames API) seems to be wrong.

**Comment by skyclouds2001** [comment]
I've updated the description and has confirmed the three APIs has different data with current one.

**Comment by Elchi3** [comment]
I think this is (mostly) done now. Lets open new issues if there are still updates to make.
